src/services/VotesService.ts

```diff
@@ -5,82 +5,27 @@ import { GetProjectVotesResponsePayload, UpsertProjectVotesRequestPayload, Votin
  * Accelerator "voting" related services
  */
 
-// TODO: remove this constant and promises below once BE is ready
-const RETURN_MOCK_VOTING_DATA = true;
-
 const ENDPOINT_VOTES = '/api/v1/accelerator/projects/{projectId}/votes';
 
 const httpVoting = HttpService.root(ENDPOINT_VOTES);
 
-let mockGetProjectVotesResponseData: VotingRecordsData;
-
 const getProjectVotes = async (projectId: number): Promise<Response2<VotingRecordsData>> =>
-  RETURN_MOCK_VOTING_DATA
-    ? new Promise((resolve) => {
-        setTimeout(() => {
-          resolve({ data: mockGetProjectVotesResponseData, requestSucceeded: true });
-        }, 300);
-      })
-    : await httpVoting.get<GetProjectVotesResponsePayload, VotingRecordsData>(
-        {
-          urlReplacements: { '{projectId}': `${projectId}` },
-        },
-        (data) => ({ votes: data?.votes })
-      );
+  await httpVoting.get<GetProjectVotesResponsePayload, Response2<VotingRecordsData>>(
+    {
+      urlReplacements: { '{projectId}': `${projectId}` },
+    },
+    (data) => ({ data: { votes: data?.votes }, requestSucceeded: true })
```

Comment:
`requestSucceeded` is set by the httpVoting.get call, you can remove it from here

Reply:
It seems like `http.get` returns a status `ok`. I think the `Response2<T>` type mandates a `requestSucceeded` flag. Maybe I am missing an object constructor for it.

Reply:
It should be set here https://github.com/terraware/terraware-web/blob/main/src/services/HttpService.ts#L85 (it's a wrapper around axios).
Are you seeing an error?

Reply:
I needed to use `get2`, which does not exist yet. I implemented it. Pushing it out now.

Reply:
I think the original `get` takes in a transform function, which we will need to pipe the variable through in the function. I implemented `get2` to return `Request2<T>` directly.

Reply:
@karthikbtf Added in the latest patch! Let me know if the approach makes sense to you.

Reply:
Tommy this looks good, thanks 👍

Reply:
We could update other similar/usages to call get2, as a separate effort.